src/pal/src/numa/numa.cpp

```diff
@@ -836,7 +836,7 @@ VirtualAllocExNuma(
             if (result != NULL && g_numaAvailable)
             {
                 int nodeMaskLength = (g_highestNumaNode + 1 + sizeof(unsigned long) - 1) / sizeof(unsigned long);
-                unsigned long *nodeMask = new unsigned long[nodeMaskLength];
+                unsigned long *nodeMask = (unsigned long*)malloc(nodeMaskLength * sizeof(unsigned long));
 
```

Comment:
Shouldn't the code check the return value of `malloc()`?

Reply:
Oh, sure it should :-(. And the other mallocs as well. Sigh.